src/main/java/view/Menu.java

```diff
@@ -0,0 +1,56 @@
+package view;
+
+import controller.Lists;
+import controller.RateList;
+import controller.StudentsOperationsClassic;
+import controller.StudentsOperationsStream;
+
+import java.io.IOException;
+import java.util.Objects;
+import java.util.Scanner;
+
+/**
+ * Created by aleksey.dobrovolsky on 6/2/2019.
+ */
+public class Menu {
+
+    public static void main(String args[]) throws IOException {
```

Comment:
StudentOperations hold reference to a static member students, hence once sorted for classics, it is sorted also for streams. Students list shouldn't be static.

Reply:
done - removed logic of static methods. The class was removed as redundant